src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Introduction to variable fonts on the web
+subhead: A new font specification that can significantly reduce font file sizes
+description: >
+  How variable fonts work, how typographers implement variable fonts, and
+  how to work with variable fonts in CSS.
+authors:
+  - mustafakurtuldu
+  - thomassteiner
+date: 2018-02-19
+updated: 2020-05-12
+hero: hero.jpg
+tags:
+  - blog
+  - fonts
+---
+
+Variable fonts can significantly reduce the size of your font files and make
+it possible to animate your font characters. This guide explains how
+variable fonts work, how typographers implement variable fonts, and how
+to work with variable fonts in CSS.
+
+## Browser compatibility {: #compatibility }
+
+As of May 2020 variable fonts are supported in most browsers. See
+[Can I use variable fonts?](https://caniuse.com/#feat=variable-fonts)
+and [Fallbacks](#fallbacks).
```

Comment:
Moved the browser compatibility data to the top of the page and updated it (it previously said VFs weren't well-supported). We know from the MDN DNA results that cross-browser compatibility is web developers number 1 frustration. So we should provide that information upfront.

Reply:
works for me